kafka_consumer/datadog_checks/kafka_consumer/client/confluent_kafka_client.py

```diff
@@ -14,15 +14,51 @@ class ConfluentKafkaClient(KafkaClient):
     @property
     def kafka_client(self):
         if self._kafka_client is None:
-            self._kafka_client = AdminClient(
-                {
-                    "bootstrap.servers": self.config._kafka_connect_str,
-                    "socket.timeout.ms": self.config._request_timeout_ms,
-                    "client.id": "dd-agent",
-                }
-            )
+            config = {
+                "bootstrap.servers": self.config._kafka_connect_str,
+                "socket.timeout.ms": self.config._request_timeout_ms,
+                "client.id": "dd-agent",
+            }
+            config.update(self.__get_authentication_config())
+
+            self._kafka_client = AdminClient(config)
+
         return self._kafka_client
 
+    def __create_consumer(self, consumer_group):
+        config = {
+            "bootstrap.servers": self.config._kafka_connect_str,
+            "group.id": consumer_group,
+        }
+        config.update(self.__get_authentication_config())
+
+        return Consumer(config)
+
+    def __get_authentication_config(self):
+        config = {
+            "security.protocol": self.config._security_protocol.lower(),
+        }
+
+        extras_parameters = {
```

Comment:
Is there an equivalent config for `tls_verify`? We originally use this option when generating the context but now have to pass everything into confluent-kafka instead.

Reply:
Having a look